-------------------------------------------------------------------------------
Test set: tech.kitucode.sockets.ClientSocketHelperTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.067 s <<< FAILURE! - in tech.kitucode.sockets.ClientSocketHelperTest
givenClientSocket(tech.kitucode.sockets.ClientSocketHelperTest)  Time elapsed: 0.023 s  <<< ERROR!
java.lang.RuntimeException: java.net.ConnectException: Connection refused
	at tech.kitucode.sockets.ClientSocketHelperTest.givenClientSocket(ClientSocketHelperTest.java:10)
Caused by: java.net.ConnectException: Connection refused
	at tech.kitucode.sockets.ClientSocketHelperTest.givenClientSocket(ClientSocketHelperTest.java:10)

